lib/index.js

```diff
@@ -25,20 +26,22 @@ function determineManifestType(filename) {
 module.exports = {
   inspect: function (root, targetFile, options) {
     options = options || {};
-    var fileContentPath = path.resolve(root || '.', targetFile || '.');
-    var manifestType;
+    let manifestType;
     try {
       manifestType = determineManifestType(path.basename(targetFile || root));
     } catch (error) {
       return Promise.reject(error);
     }
 
-    var createPackageTree = function (depTree) {
+    const createPackageTree = function (depTree) {
```

Comment:
Can we move this function out of `inspect`? I don't see any reason to define it inside.

Reply:
It needs `targetFile`. I can make it a closure.

Reply:
Or pass it as argument.